File: 2143.py
while True:
    n = int(input())
    if n == 0:
        break
    for i in range(0, n):
        a = int(input())
        if a % 2 == 0:
            print(a * 2 - 2)
        else:
            print(a*2-1)

File: 2143.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
while(true){
    let n = parseInt(lines.shift());
    if(n === 0){
        break;
    }
    for(var i = 0; i < n; i++){
        let a = parseInt(lines.shift());
        if(a % 2 === 0){
            console.log(a * 2 - 2);
        }
        else{
            console.log(a*2-1);
        }
    }
}
